src/amqproxy.cr

```diff
@@ -50,6 +52,9 @@ class AMQProxy::CLI
       parser.on("-t IDLE_CONNECTION_TIMEOUT", "--idle-connection-timeout=SECONDS", "Maxiumum time in seconds an unused pooled connection stays open (default 5s)") do |v|
         @idle_connection_timeout = v.to_i
       end
+      parser.on("--term-timeout=SECONDS", "At TERM the server will wait this many seconds for clients to gracefully close their sockets (default: infinite)") do |v|
```

Comment:
@carlhoerberg Is the text for the default-value correct (`(default: infinite)`)?
Since the if below (https://github.com/cloudamqp/amqproxy/pull/147/files#diff-60939064684e0ee6ef504d4b72e12d6cf0febd05f0426f4189b799c522816399R94) only has an effect when a value grater than 0 - which is the default - is passed.
Wouldn't the default be "zero" then, instead of "infinite"?

Reply:
changed default to -1 which now means inifite, 0 will close the app immediately